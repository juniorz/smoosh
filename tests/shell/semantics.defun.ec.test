# ADDTOPOSIX
false
f() { echo hi ; }
echo $?
f

false
f() { echo hello ; }
echo $?
f