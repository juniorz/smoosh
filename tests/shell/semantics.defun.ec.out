0
hi
0
hello
